app/assets/scripts/components/aa-table-vpromms.js

```diff
@@ -126,37 +121,30 @@ const AATable = React.createClass({
     return (
       <tbody>
         {_.map(sorted, (vpromm, i) => {
-          const vprommFieldInDB = (this.props.sources.indexOf(vpromm.id) !== -1);
-          const vprommInDB = (this.props.vpromms.indexOf(vpromm.id) !== -1);
-
-          let propBtnLabel = this.state.expandedId === vpromm.id ? 'Hide' : 'Show';
+          const vprommFieldInDB = (this.props.fieldIds.indexOf(vpromm) !== -1);
```

Comment:
For next time, you can use the Array prototype method `includes`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes